### PR TITLE
chore: Extending "@guardian/tsconfig" in tsconfig

### PR DESCRIPTION
**Pull request by akinsola-guardian**
## What does this change?
As recommended by Joe Cowton, I've installed "@guardian/tsconfig" and extended it in the tsconfig.json.
## Why?
To ensure that CMP linting standards align with the patterns recommended Client Side

### CODE REVIEW

File: cdk/tsconfig.json [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
 		"inlineSourceMap": true,
 		"inlineSources": true,
 		"experimentalDecorators": true,
 		"strictPropertyInitialization": false,
 		"typeRoots": ["./node_modules/@types"],
-		"outDir": "dist"
+		"outDir": "dist",
+		"allowJs": true,
+		"isolatedModules": true,
+		"module": "ES6",
```
Comment (sndrs) [inline cdk/tsconfig.json:26]: do you need this value specifically? the default from the config is `esnext`

Comment (akinsola-guardian) [reply cdk/tsconfig.json]: I've removed the module from this tsconfig.json file. However, I had to add the "module": "commonjs" to monitoring/tsconfig.json

File: cypress/tsconfig.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
+	"extends": "@guardian/tsconfig",
 	"compilerOptions": {
 		"esModuleInterop": true,
 		"lib": ["ES2019", "DOM"],
 		"module": "commonjs",
 		"noEmit": true,
 		// https://github.com/cypress-io/cypress/issues/9298
```
Comment (joecowton1) [inline cypress/tsconfig.json:8]: Can we remove this comment too now?

**Review by joecowton1 — APPROVED** [review]

**Review by joecowton1 — DISMISSED** [review]

**Review by sookburt — APPROVED** [review]
Looks good!

**Comment by mxdvl** [comment]
I believe this has produced an invalid output for the lib, as this version of the library leads to type errors in frontend:

**Pull request merged**